app/controllers/api/v1/graphql_controller.rb

```diff
@@ -0,0 +1,17 @@
+require "manageiq/api/common/graphql"
+
+module Api
+  module V1
```

Comment:
Shouldn't this be `V1x0` to match the version naming conventions used for the other controllers?

Reply:
it's still Api::V1x0::GraphQL   (inherited there: https://github.com/ManageIQ/catalog-api/pull/401/files#diff-2aa2a4b72cfc6cfcb6c2542db5fb91e1R10)   really common code, you can have Api::V2x0::GraphQL including the same code, it will be versioned but sharing the same base code.